src/_includes/section-menu/sdks.html

```diff
@@ -30,8 +30,27 @@
 </li>
 
 <li>
-  <span class="section-nav-list-title">DSLs</span>
+  <span class="section-nav-list-title">SQL</span>
   <ul class="section-nav-list">
-    <li><a href="{{ site.baseurl }}/documentation/dsls/sql/">SQL</a></li>
+    <li><a href="{{ site.baseurl }}/documentation/dsls/sql/overview">Overview</a></li>
+    <li><a href="{{ site.baseurl }}/documentation/dsls/sql/walkthrough">Pipeline walkthrough</a></li>
+    <li><a href="{{ site.baseurl }}/documentation/dsls/sql/cli">SQL shell</a></li>
```

Comment:
hmm, perhaps "Beam SQL shell"? or "Using the SQL shell"? and move it above the walkthrough? the thinking being, users could learn what it is and how to use it before going through the walkthrough, which I assume will use the shell.